templates/default/src/main/liberty/config/server.xml

```diff
@@ -1,13 +1,16 @@
 <server description="Liberty server">
     <featureManager>
-        <feature>microProfile-3.3</feature>
+        <feature>servlet-4.0</feature>
+        <feature>jaxrs-2.1</feature>
+        <feature>mpMetrics-2.3</feature>
+        <feature>mpHealth-2.2</feature>
+        <feature>mpConfig-1.4</feature>
     </featureManager>
     <quickStartSecurity userName="admin" userPassword="adminpwd"/>
     <keyStore id="defaultKeyStore" location="key.jks" type="jks" password="mpKeystore"/>
-    <httpEndpoint host="*" httpPort="${default.http.port}" 
-        httpsPort="${default.https.port}" id="defaultHttpEndpoint"/>
+    <httpEndpoint host="*" httpPort="${DEFAULT_HTTP_PORT}"
+        httpsPort="${DEFAULT_HTTPS_PORT}" id="defaultHttpEndpoint"/>
 
     <httpDispatcher enableWelcomePage="false" />
-    
-    <webApplication location="starter-app.war" contextRoot="/"/>
+   
```

Comment:
See the pom.xml comment for changes for server.xml

Reply:
Stuck with dropins while using the MST implied docker file. Should be able to install in apps and use web application when we have a custom docker file for MST.

Reply:
So if we commit our own Dockerfile can we add the apps config?

Reply:
Yes